plugins/epos/config.ini

```diff
@@ -24,6 +24,7 @@ signposting = Signposting
 # Metadata terms to find the resource identifier
 identifier_term = [['id'],]
 
+
```

Comment:
remove

Reply:
Are you referring to identifier term? That is used in some tests , is the one that contains the UUID

Reply:
no, remove the new line, or has this been done by black formatter?